corehq/apps/importer/util.py

```diff
@@ -95,6 +97,25 @@ def from_request(cls, request):
 
         )
 
+
+def open_workbook(filename=None, **kwargs):
+    _soft_assert = soft_assert(notify_admins=True)
+    try:
+        return xlrd.open_workbook(filename=filename, **kwargs)
+    except xlrd.XLRDError as e:
+        message = unicode(e)
+        if message == u'Workbook is encrypted':
+            raise ImporterExcelFileEncrypted(message)
+        else:
+            _soft_assert(False, 'displaying XLRDError directly to user', e)
+            raise ImporterExcelError(message)
```

Comment:
I realize this is behind a soft assert, so maybe not too important, but this will create a new traceback starting from this point, which may be harder to debug. To work around this issue you could do:
```
except xlrd.XLRDError as e:
    exc_type, exc_value, exc_tb = sys.exc_info()
    if ...
    else:
        raise exc_type, exc_value, exc_tb # reraise original exception